### DISCUSSION TITLE
filtering osu!mania beatmaps by key-types

### DISCUSSION THREAD

**Discussion by gkstmdwn** [ppy/osu#22700] [Ideas] [answered]
What about adding a setting for osu!mania users where they can filter beatmaps by selecting the key types?
For example, If you select 4keys in the filtering, you would only see 4key beatmaps.

**Comment by bdach** [comment, marked as answer]
This is already partially possible if you type `keys=4` in the search box in song select, but it won't correctly work for converts yet.

**Reply by gkstmdwn** [reply]
Oh sorry, I should have searched more for it... thanks
